### ISSUE TITLE
Nvidia driver 545 from Pop Shop caused screen to go black, had to revert to 470

### ISSUE THREAD

**Issue by rf300400** [pop-os/pop#3188]
Distribution:

NAME="Pop!_OS"
VERSION="22.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 22.04 LTS"
VERSION_ID="22.04"
VERSION_CODENAME=jammy
UBUNTU_CODENAME=jammy

  
Related Application and/or Package Version :

nvidia-driver-545:
  Installed: (none)
  Candidate: 545.29.06-1pop0~1701107297~22.04~7642405
  Version table:
     545.29.06-1pop0~1701107297~22.04~7642405 1001
       1001 http://apt.pop-os.org/release jammy/main amd64 Packages

nvidia-driver-470:
  Installed: 470.199.02-1pop0~1695931040~22.04~4c2d0ee
  Candidate: 470.199.02-1pop0~1695931040~22.04~4c2d0ee
  Version table:
     470.223.02-0ubuntu0.22.04.1 500
        500 http://apt.pop-os.org/ubuntu jammy-security/restricted amd64 Packages
        500 http://apt.pop-os.org/ubuntu jammy-updates/restricted amd64 Packages
 *** 470.199.02-1pop0~1695931040~22.04~4c2d0ee 1001
       1001 http://apt.pop-os.org/release jammy/main amd64 Packages
        100 /var/lib/dpkg/status
     470.103.01-0ubuntu2 500
        500 http://apt.pop-os.org/ubuntu jammy/restricted amd64 Packages

Issue/Bug Description: Nvidia driver 545 from Pop Shop caused screen to go black, had to revert to 470

Steps to reproduce : Updating to nvidia-driver-545

Expected behavior: Everything going on as usual

Other Notes:
Machine: Type: Laptop System: HP product: Victus by HP Laptop 16-e0xxx v: N/A

CPU:
  Info: 6-core model: AMD Ryzen 5 5600H with Radeon Graphics bits: 64
    type: MT MCP cache: L2: 3 MiB
  Speed (MHz): avg: 915 min/max: 400/4280 cores: 1: 400 2: 400 3: 3295
    4: 400 5: 400 6: 400 7: 400 8: 400 9: 400 10: 400 11: 400 12: 3695
Graphics:
  Device-1: NVIDIA TU117M [GeForce GTX 1650 Mobile / Max-Q] driver: nvidia
    v: 470.199.02
  Device-2: AMD Cezanne driver: amdgpu v: kernel
  Device-3: Luxvisions Innotech HP Wide Vision HD Camera type: USB
    driver: uvcvideo
  Display: x11 server: X.Org v: 1.21.1.4 driver: X:
    loaded: amdgpu,ati,nvidia unloaded: fbdev,modesetting,nouveau,radeon,vesa
    gpu: amdgpu resolution: 1920x1080~60Hz
  OpenGL:
    renderer: RENOIR (renoir LLVM 15.0.7 DRM 3.54 6.5.6-76060506-generic)
    v: 4.6 Mesa 23.2.1-1pop0~1699535617~22.04~6cd9a72

**Comment by rf300400** [comment]
Apparantly I just had to alt f2 and then raise skinny elephants. The following restart caused my laptop to get back to normal.

**Comment by philg-dev** [comment]
Note for anybody who doesn't know what "raise skinny elephants" refers to:
https://www.hashbangcode.com/article/raising-skinny-elephants-utterly-boring

It's basically a last resort for restarting your machine when it freezes / locks up when your only other option would be to pull the plug.

I just had the black screen problem when I wanted to install NVIDIA Driver 555 from the Pop Shop, since my system was running version 470, despite the fact that I'm not on a Legacy GPU.

To sum up my case for others that might encounter the same:
- Lenovo Legion Laptop with NVIDIA RTX 2060
- I chose to install NVIDIA Driver 555 via Pop Shop
- After the download was finished and the actual installation process began both of my screens turned black
- the laptop was clearly still working on it (fans were fairly loud)
- a few minutes later fans went more quiet again, as in system idle
- another few minutes later I did the "Raising skinny elephants is utterly boring" thing:
    - `ALT + SysRq + rseiub` (in my case `SysRq = PrintScrn` key)
- the system rebooted and the 555 driver was working properly

Hope this might help someone in the future.
